doc/modules/sortingcomponents.rst

```diff
@@ -223,9 +223,9 @@ Here is a short example the depends on the output of "Motion estimation":
 
 **Notes**:
   * :code:`spatial_interpolation_method` "krigging" or "iwd" fot not play a big role.
-  * :code:`border_mode` is very imprtant details. How to deal with the border because with motion untis on the border
+  * :code:`border_mode` is very important details. How to deal with the border because with motion units on the border
```

Comment:
```suggestion
  * :code:`border_mode` is a very important detail. It controls how to deal with the border because with motion units on the border
```

Reply:
I tried rewriting this one for clarity so definitely give it a look over to see if you like how I rewrote it.